### DISCUSSION TITLE
Adjustable edge paths

### DISCUSSION THREAD

**Discussion by WillGuima** [xyflow/xyflow#2708] [New Features]
### Please describe the feature that you want to propose

Thanks for the great tool you have been building.
Is adjustable edge paths on the roadmap?

Example:

https://user-images.githubusercontent.com/28910003/208657440-56a5b837-9635-4fa8-a784-74ec66ed5b43.mov

**Comment by ansaganie** [comment]
Hello guys,

Does anybody have an example of the implementation of this kind of edge? Or maybe any ideas on how it could be implemented?
